frappe/database/query.py

```diff
@@ -50,10 +51,15 @@ def get_query(
 		validate_filters: bool = False,
 		skip_locked: bool = False,
 		wait: bool = True,
+		ignore_permissions: bool = True,
```

Comment:
So, there aren't THAT many uses of this in the wild. Once this is stable enough, we should break this and flip the defaults. 

ref: https://sourcegraph.com/search?q=context:global+frappe.qb.get_query+repo:%5Egithub%5C.com/frappe/.*&patternType=standard&sm=0